src/DomCrawler/Crawler.php

```diff
@@ -229,7 +229,7 @@ public function html(string $default = null): string
                 return $this->webDriver->getPageSource();
             }
 
-            return $this->attr('outerHTML');
+            return $this->attr('outerHTML', $default);
```

Comment:
```suggestion
            return $this->attr('outerHTML', (string) $default);
```

Because we use strict types in Panther's code base.

Reply:
@dunglas Done ;)